mfr/extensions/tabular/static/js/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
Only the css. A slightly older version of bootstraps' css is being used for displaying certain error messages, but assuming PR #186 gets merged, otherwise no. 

I can refactor it to be globally usable, but my thought was that since this is the only extension using bootstrap.js, keeping it here until it's used by other extensions (if that happens) could slightly reduce bloat, as not every instance of the MFR would need to have every extension installed.

Also, seeing as how only two css classes are being used in the error messages, it might make sense to remove that dependency and use only those classes (similar to the rationale behind #186 ).